app/src/main/java/unipd/se18/ocrcamera/EANResolve.java

```diff
@@ -0,0 +1,27 @@
+package unipd.se18.eanresolvemodule;
+
+/**
+ * Useful set a type of recognition
+ * It avoids the single point of failure relative to this process
+ * @author Elia Bedin
+ */
+public class EANResolve {
+    /**
+     * Ids of the different EAN resolver API
+     */
+    public enum API {
+        mignify
```

Comment:
Enum cases are usually uppercased

Reply:
Thank you for pointing this out, I definitely missed that re-checking my code!